From 134feb6e889eaa6c06316e89c9af903de658bfcc Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Thu, 21 Nov 2019 00:29:25 +0200
Subject: [PATCH 2/2] layer: Add bbmask for Boot2Qt

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 conf/layer.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/conf/layer.conf b/conf/layer.conf
index 1bc9d54..0d7dd64 100644
--- a/conf/layer.conf
+++ b/conf/layer.conf
@@ -10,5 +10,6 @@ BBFILE_PATTERN_compulab-bsp := "^${LAYERDIR}/"
 BBFILE_PRIORITY_compulab-bsp = "9"
 
 BBMASK += "meta-compulab/recipes-xfce"
+BBMASK += "meta-bsp-imx8mm/recipes-bsp/bt-start"
 
 LAYERSERIES_COMPAT_compulab-bsp = "sumo"
-- 
2.11.0

